### PR TITLE
feat: create a header component

### PR DESCRIPTION
**Pull request by roxanehe**
### Fixes
created first version of header component

### Testing Instructions

'npm run dev' to see the project
'npm run storybook' to view storybook page

### Screenshots 
image
image

### CODE REVIEW

**Comment by roxanehe** [comment]
errors are gone, ready to be reviewed

File: frontend/package.json [hunk 1]
```diff
@@ -31,9 +32,11 @@
     "@storybook/vue3": "^7.0.7",
     "@storybook/vue3-vite": "^7.0.7",
     "@types/jsdom": "^21.1.0",
-    "@types/node": "^18.14.2",
+    "@types/node": "^18.16.3",
     "@vitejs/plugin-vue": "^4.0.0",
     "@vitejs/plugin-vue-jsx": "^3.0.0",
+    "@vue/cli-plugin-typescript": "^5.0.8",
+    "@vue/cli-service": "^5.0.8",
```
Comment (YoussefLaunchUp) [inline frontend/package.json]: What are these libraries for, are they required?

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
There are eslint errors that need to get fixed
image

Also the PR title needs to be using conventional commits
more info: https://www.conventionalcommits.org/en/v1.0.0-beta.2/

File: frontend/src/components/Title.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="headerTitle">
```
Comment (neli-rashkova) [inline frontend/src/components/Title.vue]: Good work so far on the first version of the header. Just wondering what is the class `headerTitle` referring to?

File: frontend/package.json (hunk 1 above)
Comment (roxanehe) [reply frontend/package.json]: deleted unnecessary libraries and fixed Eslint issues

File: frontend/src/components/Title.vue (hunk 2 above)
Comment (roxanehe) [reply frontend/src/components/Title.vue]: oh it actually has no effects, just added it as if a convention in css lol, deleted it

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
Looks like the Text and the image are not aligned well. I would also add some padding on the left of the screen 

image

**Review by simsung1 — COMMENTED** [review]
Looks good to me! Just pending the changes that Youssef requested, then I think you're good to merge after that.

**Comment by roxanehe** [comment]
> Looks like the Text and the image are not aligned well. I would also add some padding on the left of the screen
> 
> image

fixed it!
image

**Review by YoussefLaunchUp — APPROVED** [review]
Great work

**Review by MjxOro — APPROVED** [review]
LGTM!

**Pull request merged**